src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/ccd/datamigration/controller/CcdDataMigrationControllerTest.java

```diff
@@ -50,6 +56,19 @@
     private static final String MIGRATE_FRC_URL = "/ccd-data-migration/migrateFrc";
     private static final String MIGRATE_URL = "/ccd-data-migration/migrate";
 
+    @Autowired
+    CcdDataMigrationController CcdDataMigrationController;
+
+    @MockBean
+    RemoveRespondentSolOrg removeRespondentSolOrg;
+
+    @Test
+    public void shouldRemoveRespOrgPolicyFromCaseData() {
+        CcdDataMigrationController.migrate(authTokenGenerator.generate(), buildCallbackRequest());
+
+        verify(removeRespondentSolOrg, times(1)).migrateCaseData(any());
```

Comment:
rather than any can you validate that we pass the case data from buildCallbackRequest()